source/layouts/layout.erb

```diff
@@ -1,69 +1,59 @@
+                <image src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image"></image>
+              </svg>
+              <span class="govuk-header__logotype-text">
+                GOV.UK
+              </span>
             </span>
-            <span class="header__title">
+            <span class="govuk-header__product-name">
               Product Pages
-              <span class="phase-banner">Beta</span>
+              <strong class="govuk-tag govuk-phase-banner__content__tag">
+                beta
+              </strong>
             </span>
           </a>
         </div>
-
-        <div data-module="navigation">
-          <button type="button" class="header__navigation-toggle js-nav-toggle" aria-controls="navigation" aria-label="Show or hide top level navigation">Menu</button>
-
-          <nav id="navigation" class="header__navigation js-nav" aria-label="Top Level Navigation" data-click-events data-click-category="Header" data-click-action="Navigation link clicked">
-            <ul>
+        <div class="govuk-header__content">
+          <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+          <nav>
+            <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
               <%
                 {
                   'About' => '/',
-                  'Supplementary Pages' => '/supplementary.html',
-                  'Support' => '/support.html'
+                  'Supplementary Pages' => '/supplementary',
+                  'Support' => '/support'
```

Comment:
This active state logic only works for the first "About" link because the `current_page.url` includes the trailing slash. I've added the slashes locally and it does fix the problem.

I don't know how to do the GitHub suggestion-comment thing so here's a patch if you want to use that:

```patch
From fe6d950cb40498be753e386c28c87b1c8dd5c8c9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Maniaci?=
 <stephane.maniaci@digital.cabinet-office.gov.uk>
Date: Wed, 28 Aug 2019 14:25:12 +0100
Subject: [PATCH] Fix the active menu item logic

`current_page.url` includes the trailing slash so it needs to be
present in the menu map as well.
---
 source/layouts/layout.erb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/layouts/layout.erb b/source/layouts/layout.erb
index d0402bb..bbcd6b4 100644
--- a/source/layouts/layout.erb
+++ b/source/layouts/layout.erb
@@ -48,8 +48,8 @@
               <%
                 {
                   'About' => '/',
-                  'Supplementary Pages' => '/supplementary',
-                  'Support' => '/support'
+                  'Supplementary Pages' => '/supplementary/',
+                  'Support' => '/support/'
                 }.each do |title, url|
               %>
                 <li class="govuk-header__navigation-item<% if url == current_page.url %> govuk-header__navigation-item--active" aria-current="page<% end %>">
-- 
2.17.2 (Apple Git-113)
```

Reply:
ah good spot